### PR TITLE
Bug fixes for disabled addressbar participants text

### PR DESCRIPTION
**Pull request by artonragsdale**
There are a few bugs in the generation of the participants text of a disabled address bar, which this pull request fixes.

• The plural string for 'other' was not set correctly (I added an 's' to the string)
• If a single participant's name was too long to display, the resulting string was '1 participants' (This was apparent due to the iOS 9 bug to follow. I checked the number of participants and used singular if it was only 1)
• When the 'n participants' string was used, the number of participants was incorrectly calculated. (Caused by decreasing the count for appending remaining participants and using the same count when no participants are named)
• On iOS 9, the space available for the string was being calculated before the view frames were established, resulting in the participant names never being displayed (I fixed this by setting the disabled text in the viewDidLayoutSubviews method.

### CODE REVIEW

File: Code/Controllers/ATLAddressBarViewController.m [hunk 1]
```diff
@@ -478,23 +486,35 @@ - (NSString *)disabledStringForParticipants:(NSOrderedSet *)participants
...
-    NSString *othersString = (remainingParticipants > 1) ? ATLLocalizedString(@"atl.addressbar.others.key", @"other", nil) : ATLLocalizedString(@"atl.addressbar.other.key", @"other", nil);
+    NSString *othersString = (remainingParticipants > 1) ? ATLLocalizedString(@"atl.addressbar.others.key", @"others", nil) : ATLLocalizedString(@"atl.addressbar.other.key", @"other", nil);
     return [NSString stringWithFormat:@"%@ %lu %@", ATLLocalizedString(@"atl.addressbar.and.key", @"and", nil), (unsigned long)remainingParticipants, othersString];
 }
 
 - (BOOL)textViewHasSpaceForParticipantString:(NSString *)participantString
 {
     CGSize fittedSize = [participantString sizeWithAttributes:@{NSFontAttributeName: self.addressBarView.addressBarTextView.font}];
-    return fittedSize.width < (CGRectGetWidth(self.addressBarView.addressBarTextView.frame) - ATLAddressBarTextViewIndent - ATLAddressBarTextContainerInset - ATLDisabledStringPadding); // Adding extra padding to account for text container inset.
+    CGFloat textViewWidth = CGRectGetWidth(self.addressBarView.addressBarTextView.frame);
```
Comment (kmahal) [inline Code/Controllers/ATLAddressBarViewController.m]: do your changes here change any functionality or simply space it out?

File: Code/Controllers/ATLAddressBarViewController.m [hunk 2]
```diff
@@ -478,23 +486,35 @@ - (NSString *)disabledStringForParticipants:(NSOrderedSet *)participants
...
+            //Increase participant count since we're not using the first participant's name
+            remainingParticipants++;
+            disabledString = [self stringForParticipantCount:remainingParticipants];
             *stop = YES;
         }
     }];
     return disabledString;
 }
 
+- (NSString *)stringForParticipantCount:(NSInteger)participantCount
```
Comment (kmahal) [inline Code/Controllers/ATLAddressBarViewController.m]: use `NSUInteger`

File: Code/Controllers/ATLAddressBarViewController.m [hunk 3]
```diff
@@ -72,6 +72,15 @@ - (void)viewDidLoad
     [self configureLayoutConstraintsForTableView];
 }
 
+- (void)viewDidLayoutSubviews{
+    
+    if (self.disabled == YES) {
+        //set disabled text after views have been laid out
+        self.addressBarView.addressBarTextView.text = [self disabledStringForParticipants:self.selectedParticipants];
+    }
+
```
Comment (kmahal) [inline Code/Controllers/ATLAddressBarViewController.m]: remove extra line

File: Code/Controllers/ATLAddressBarViewController.m [hunk 4]
```diff
@@ -72,6 +72,15 @@ - (void)viewDidLoad
     [self configureLayoutConstraintsForTableView];
 }
 
+- (void)viewDidLayoutSubviews{
```
Comment (kmahal) [inline Code/Controllers/ATLAddressBarViewController.m]: add `{` to the next line

File: Code/Controllers/ATLAddressBarViewController.m [hunk 5]
```diff
@@ -72,6 +72,15 @@ - (void)viewDidLoad
...
+
```
Comment (kmahal) [inline Code/Controllers/ATLAddressBarViewController.m]: remove extra line

**Comment by kmahal** [comment]
Thanks @artonragsdale for your PR.  I've added a few comments.  In addition, please squash your commits down to 1 commit, as 10 commits including a few merge commits are excessive for the functionality change.  Thank you for the work!

File: Code/Controllers/ATLAddressBarViewController.m (hunk 1 above)
Comment (artonragsdale) [inline Code/Controllers/ATLAddressBarViewController.m]: It does not change functionality. It was modified to make it easier to debug.

File: Code/Controllers/ATLAddressBarViewController.m [hunk 6]
```diff
@@ -72,6 +72,15 @@ - (void)viewDidLoad
     [self configureLayoutConstraintsForTableView];
 }
 
+- (void)viewDidLayoutSubviews{
+    
+    if (self.disabled == YES) {
```
Comment (kmahal) [inline Code/Controllers/ATLAddressBarViewController.m]: no need for `== YES`, just keep it `if (self.disabled)`

**Comment by artonragsdale** [comment]
I have created a new pull request with your desired changes and have closed this request.

**Pull request closed without merging**